src/core/core.defaults.js

```diff
@@ -39,13 +57,63 @@ export class Defaults {
 		this.title = undefined;
 		this.tooltips = undefined;
 		this.doughnut = undefined;
+		this._routes = {};
 	}
 	/**
 	 * @param {string} scope
 	 * @param {*} values
 	 */
 	set(scope, values) {
-		return merge(this[scope] || (this[scope] = {}), values);
+		return merge(getScope(this, scope), values);
+	}
+
+	/**
+	 * Routes the named defaults to fallback to another scope/name.
+	 * This routing is useful, when those target values, like defaults.color, is changed runtime.
```

Comment:
nit:
This routing is useful when those target values, like defaults.color, are changed runtime.